ci/Dockerfile

```diff
@@ -1,46 +1,131 @@
-FROM a12e/docker-qt:5.14-gcc_64
+ARG QT_VERSION=5.15.2
+
+# QT Installation Image --------------------------------------------------------
+FROM ubuntu:18.04 AS qt-install
```

Comment:
Why don't we use something more recent here? That would allow us to use the latest Qt 5.15.x version

Reply:
Heck, why do we even use Ubuntu in the first place? ;)

Reply:
ubuntu-latest (stable) points to https://hub.docker.com/layers/library/ubuntu/latest/images/sha256-965fbcae990b0467ed5657caceaec165018ef44a4d2d46c7cdea80a9dff0d1ea?context=explore

Reply:
aka 22.04

Reply:
Yes, 22.04 would be nice. If we don't need to support older linux versions we could go with the Nix version @jakubgs tested here https://github.com/status-im/status-desktop/pull/9232 and get rid of Docker and Ubuntu. 🚀

Reply:
can we bundle glibc in the AppImage? :imp:

Reply:
Hmm, I see it has lots of dependencies and we might be forced to pack a large dependency tree.
https://www.linux.co.cr/ldp/lfs/appendixa/glibc.html

Reply:
We can't because we can't use new glibc. We had this conversation many times before. If you want to use newer version of glibc you'll have to convince @iurimatias that we shouldn't support older linux installations.

Reply:
The alternative to that is a [glibc header hack](https://github.com/wheybags/glibc_version_header), but I never tried making that work.

Reply:
I see... that's unfortunate; the other solution would be to have 2 docker environments right? (older/newer)

Reply:
One other option would be to build Qt from source for our builds.

Reply:
This wouldn't solve the glibc problem, right?

Reply:
It would if we build it on an older linux version.

Reply:
Would QT version built on old glibc alone be enough to fix this. I don't think that's the issue here, I think it's the underlying system that causes this, not QT.

Reply:
The only reason why I used Qt5.15.2 + KDAB instead of Qt5.15.8 + KDAB is because the newer qt is built against a newer glibc. I've tried the homebrew qt@5 package that is a Qt5.15.8 + KDAB and could not use it due to the glibc dependency. This makes me think that if we build Qt using Ubuntu 18.04 we could use the newer version.

Reply:
But how does that make sense? If the issue is the Glibc verison with which QT was built the our underlying system doesn't matter, because we don't build Qt, we install a distribution via `aqtinstall`.

Reply:
If what you're saying is the case then we don't have to actually use Ubuntu 18.04.

Reply:
linuxdeployqt [will not work](https://github.com/probonopd/linuxdeployqt#a-note-on-binary-compatibility) on Ubuntu newer than 18.04.

Reply:
@alexjba what "+ KDAB" means?

Reply:
I see, if the issue is also with `linuxdeployqt` that's also worth documenting.

Reply:
> @alexjba what "+ KDAB" means?

With kdab patches.

> linuxdeployqt [will not work](https://github.com/probonopd/linuxdeployqt#a-note-on-binary-compatibility) on Ubuntu newer than 18.04.

Looks like it will be a real effort to support older linux systems. We might need to build linuxdeployqt also.